lblod/ember-rdfa-editor-lblod-plugins
# Redesign structure nodes v1

### Overview
Added some css to match Saska design https://www.figma.com/design/tfBSv7999dzz1sy6svMfsY/GN_editor?node-id=81-575&node-type=frame&t=GxqZjDXLUvDzepry-0

##### connected issues and PRs:
GN-5118


### Setup
None

### How to test/reproduce
Insert an article and see the design, you can also inserte several articles one inside another 

### Challenges/uncertainties
We will have to recheck the padding on the left when adding the buttons



### Checks PR readiness
